configs/mx7ulpea-ucom_lmp_defconfig

```diff
@@ -21,6 +21,7 @@ CONFIG_BOOTCOMMAND="fatload mmc 0:1 ${loadaddr} /boot.itb; setenv verify 1; sour
 CONFIG_DEFAULT_FDT_FILE="imx7ulpea-ucom-kit_v2"
 CONFIG_BOUNCE_BUFFER=y
 CONFIG_BOARD_EARLY_INIT_F=y
+CONFIG_FIOVB=y
```

Comment:
Please also add the same changes to configs/mx7ulpea-ucom-mfgtool_defconfig, as that is the one used by the mfgtool build.